pkg/services/alertmanager.go

```diff
@@ -33,13 +33,17 @@ type AlertmanagerNotification struct {
 
 // AlertmanagerOptions cluster configuration
 type AlertmanagerOptions struct {
-	Targets            []string   `json:"targets"`
-	Scheme             string     `json:"scheme"`
-	APIPath            string     `json:"apiPath"`
-	BasicAuth          *BasicAuth `json:"basicAuth"`
-	BearerToken        string     `json:"bearerToken"`
-	InsecureSkipVerify bool       `json:"insecureSkipVerify"`
-	Timeout            int        `json:"timeout"`
+	Targets             []string   `json:"targets"`
+	Scheme              string     `json:"scheme"`
+	APIPath             string     `json:"apiPath"`
+	BasicAuth           *BasicAuth `json:"basicAuth"`
+	BearerToken         string     `json:"bearerToken"`
+	Timeout             int        `json:"timeout"`
+	InsecureSkipVerify  bool       `json:"insecureSkipVerify"`
+	MaxIdleConns        int        `json:"maxIdleConns"`
```

Comment:
I think it makes sense to create a struct that holds all transport related settings:

```golang
type TransportOptions struct {
	MaxIdleConns        int         `json:"maxIdleConns"`
	MaxIdleConnsPerHost int         `json:"maxIdleConnsPerHost"`
	MaxConnsPerHost     int         `json:"maxConnsPerHost"`
	IdleConnTimeout     string      `json:"idleConnTimeout"`
}

type AlertmanagerOptions struct {
...
	TransportOptions
}

httputil.NewServiceHTTPClient(s.opts.TransportOptions)
```

This way we can add more options in a future without touching all services